.pre-commit-config.yaml

```diff
@@ -9,6 +9,21 @@ repos:
     hooks:
     -   id: isort
         name: isort
+-   repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v1.1.1
+    hooks:
+      - id: mypy
+        exclude: ^tests/
```

Comment:
it would be great to put a regex that would say "tests/" or "api/migrations" (or at least "api/migrations") but I can't figure out how to do it :)

Reply:
It should be regexp, I think something like 
`exclude: "(^tests/)|(api/migrations)|"` etc.

Reply:
Also I think migrations here are excluded by default (you could check it by removing exclude both from this file and pyproject.toml)
 
I think it's because of mypy django plugin

Reply:
Oh, great. As for the regex, I guess `^` should be before the opening `(`, but never mind then :)